sajibportfolio.html

```diff
@@ -1,273 +1,146 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-  <meta charset="UTF-8" />
-  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <link rel="stylesheet" type="text/css" href="sajibportfolio.css" />
-  <title>Document</title>
-</head>
-
-<body>
-  <section id="works">
-    <section class="mwp1">
-      <div class="sp1"></div>
-      <div class="lb2">
-        <div class='pt2'>
-          <h2 class="prt2">Multi-Post
-            <br /> Stories
-          </h2>
-          <div class="fr2">
-            <div class="tl">
-              <p class="tlt">CANOPY</p>
-            </div>
-            <span class="ct"></span>
-            <!-- <img class="counter" src="statics/images/Counter.png" alt="counter"/> -->
-            <div class="jt">
-              <p class="jtt">Back End Dev</p>
-            </div>
-            <span class="ct1"></span>
-            <!--<img class="counter" src="statics/images/Counter.png" alt="counter">-->
-            <div class="yr">
-              <p class="yrt">2015</p>
-            </div>
-          </div>
-        </div>
-        <p class="jhora">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-        <ul class="tgs">
-          <li class="li1">
-            <p class="lp1">html </p>
-          </li>
-          <li class="li2">
-            <p class="lp2">css</p>
-          </li>
-          <li class="li3">
-            <p class="lp3">javascript</p>
-          </li>
-        </ul>
-        <div class="btl">
-          <button class="btn"><a href="#" class="btt">See Project</a></button>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="/sajibportfolio.css" />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <title>Finish Mobile Version</title>
+  </head>
+
+  <body>
+    <section id="works">
+      <section id="my_work">
+        <div class="snapshot_portfolio"></div>
+          <div id="ptext">
+              <h1 class="ptitle">Sajib</h1>
+              <ul class="tags">
+                <li>
+                  <p>CANOPY</p>
+                </li>
+                <li>
+                  <p>Back End Dev</p>
+                </li>
+                <li>
+                  <p>2015</p>
```

Comment:
- [ ] You did a great job so far, I can see that when running your project is not really complete, your header section is not displaying in the browser, we are just seeing this part :point_down: 
![Screenshot from 2022-12-07 02-04-13](https://user-images.githubusercontent.com/88894525/206053002-697af86b-420d-44dc-b11e-a92460ef38f2.png)

Reply:
completed.